docs/devel/RELEASE.md

```diff
@@ -76,35 +80,43 @@ CHANGELOG.md snippet and uploading the binaries from the build phase.
 	git diff
 	```
 
-1. Commit the changes and push to remote repository.
+1. Commit the changes and push to remote repository fork.
 
 	```
 	git commit -a -s -m "Prepare v<next_semvar> development"
 	git push origin master
 	```
 
-## Building
-
-1. Obtain copy of remote repository.
+1. Create a Pull Request for the development preparation commit.
 
-	```
-	git clone git@github.com:open-policy-agent/opa.git
-	```
+## Publishing
 
-1. Execute the release target. The results can be found under _release/VERSION:
+1. Push the release tag to remote source repository.
 
 	```
-	make release VERSION=0.12.8
+	git push upstream v<semver>
 	```
 
-## Publishing
+   > Note: Only OPA maintainers will have permissions to perform this step.
+
+1. Open browser and go to [https://github.com/open-policy-agent/opa/releases](https://github.com/open-policy-agent/opa/releases)
 
-1. Open browser and go to https://github.com/open-policy-agent/opa/releases
+1. Update the draft release (may take up to 20 min for the draft to become
+   available, track its process under
+   [https://github.com/open-policy-agent/opa/actions](https://github.com/open-policy-agent/opa/actions)).
+   Ensure everything looks OK and publish when ready.
 
-1. Create a new release for the version.
-	- Copy the changelog content into the message.
-	- Upload the binaries.
+### Optional Publishing Method
```

Comment:
What's the scenario where we'd want or need to do this? I'd rather just omit this from the doc since there are a bunch of caveats (title must match tag, do not upload binaries, etc).

Reply:
Good question.. I guess I documented it mostly to inform if someone _did_ want to try and do it through the UI for whatever reason (maybe you gotta cut the release on your phone?).

I'm open to suggestions for this, we could maybe rename it from `Optional Publishing Method` to like `Via GitHub UI (Not Recommended)`

Reply:
I'd just delete it. More options = more decisions and more room for error. I feel like less is more in this case.

Reply:
Done